docs/photon/deployment.md

```diff
@@ -2,10 +2,40 @@
 
 Photon.js depends on a query engine that's running as a binary on the same host as your application. When deploying your Photon-based application to production, you need to ensure that the binary used by Photon can run in your production environment, i.e. it needs to be compatible with the runtime of your deployment provider.
 
-The query engine binary is downloaded when you run `prisma2 generate`, it is then stored alongside the generated Photon code inside `node_modules/@generated` (or the [custom `output` path](./codegen-and-node-setup.md) you specified). 
+The query engine binary is downloaded when you run `prisma2 generate`, it is then stored alongside the generated Photon code inside `node_modules/@generated` (or the [custom `output` path](./codegen-and-node-setup.md) you specified).
 
 **IMPORTANT**: To ensure the query engine binary is compatible with your production environment, you have to [specify the right platform for Photon.js](../core/generators/photonjs.md#specifying-the-right-platform-for-photon-js).
 
+## Photon in FaaS environment (Like AWS Lambda)
+
+**DB Connection Handling**
+
+Nuances around handling DB connections in Lambda are not new and most of those nuances also apply to Photon.
+
+Lambda has the concept of [reusing a container](https://aws.amazon.com/blogs/compute/container-reuse-in-lambda/) which means that for subsequent invocations of
+the same function it may use an already existing container that has the allocated processes, memory, file system (`/tmp` is writable in Lambda), and even DB
+connection still available.
+
+Any piece of code [outside the handler](https://docs.aws.amazon.com/lambda/latest/dg/programming-model-v2.html) remains initialized. This is a great place for
+`Photon` to call "connect" or at least call `Photon` constructor so that subsequent invocations can share a connection. There are some implications though they
+are not directly related to Photon but any system that would require a DB connection from Lambda:
+
+| Implication                                                                                                                                                                                                                                                                                                                           | Potential Solution                                                                                                                                               |
```

Comment:
I'm not sure if a table is the best element to display the information here, right now there's no easy way for a reader to get a _quick_ overview of all the implications. They always need to read the entire text in the table. I'd suggest to use another element (or restructure the table) to make this easier to consume. Two ideas:

- Instead of a table, use subheadings that briefly describe the implication in a few words and then use the text you  wrote for the description in regular paragraphs, maybe prepend each paragraph with **Description** and **Potential solution** in bold (that way readers can grasp all implications by just scanning the page and read more about one by just reading the paragraphs below the headline)
- If you want to keep using the table, add another column so that we have three columns, e.g. `Implication`, `Description`, `Potential solution`. That way you can use the first column to again briefly describe the implication in a few words, then just reuse the current two columns for `Description` and `Potential solution`.

Reply:
Went with table + extra column, let me know if you want that changed :)